app/views/layouts/application.html.erb

```diff
@@ -33,7 +35,7 @@
   </head>
 
   <body class="<%= controller_name %> <%= action_name %>">
-
+    <%= render partial: "layouts/tag_manager_snippet" %>
```

Comment:
Same as above: Did you mean to add this the new flow? https://github.com/openstax/accounts/blob/master/app/views/layouts/newflow_layout.html.erb

Only adding it to app/views/layouts/application.html.erb will make it only available in the legacy flow and users' profile.

Reply:
Fixed